scripts/targetwindow.lic

```diff
@@ -47,14 +49,19 @@ module TargetWindow
     old_targets = GameObj.targets
     _respond("<closeDialog id='TargetWindow'/><openDialog type='dynamic' id='TargetWindow' title='Targets' target='TargetWindow' scroll='manual' location='main' justify='3' height='68' resident='true'><dialogData id='Targets'></dialogData></openDialog>")
     loop {
-      if GameObj.targets.length > 0
+      targets = GameObj.targets.select { |t| !(t.noun =~ @grasp_arms || t.name =~ /^animated /) }
+      if targets.length > 0
         output = "<dialogData id='TargetWindow' clear='t'>"
-        output += "<link id='total' value='Total Targets: #{GameObj.targets.delete_if { |t| t.noun =~ @grasp_arms }.length}' cmd='target next' echo='target next' justify='3' top='3' left='0' height='15' width='195'/>"
-        GameObj.targets.each { |t|
-          next if t.noun =~ @grasp_arms
+        output += "<link id='total' value='Total Targets: #{targets.length}' cmd='target next' echo='target next' justify='3' top='3' left='0' height='15' width='195'/>"
+        targets.each { |t|
           target_status = TargetWindow.status_fix(t.status) unless t.status.nil?
           output += "<link id='#{t.id}' value='#{target_status} #{t.name.split.map(&:capitalize).join(' ')}' cmd='target ##{t.id}' echo='target ##{t.id}'  justify='3' left='0' height='15' width='195'/>"
         }
+        if CharSettings['show_arms']
+          arms = GameObj.targets.select { |t| t.noun =~ @grasp_arms }
+          output += "<label id= 'armCount' value='Total arms: #{arms.length}' top='#{15 * (targets.length + 2)}' left='0'  width='187'/>" unless arms.empty?
```

Comment:
The "top" section is basically dynamically adding 2 lines of space for the "Total arms:" after the mob list.